docs/Developers/Review.rst

```diff
@@ -83,10 +83,11 @@ The review process for coala is as follows:
     .. note::
 
         Only one acknowledgment is needed per commit i.e ``ack commit_SHA``.
-
-4. If the commits are not linearly mergeable into master, rebase and go
+4. After the PR has been reviewed, comment on a review unless there is a technical
```

Comment:
I guess you meant to say `don't comment on a review unless ...`

Reply:
Made it more clear.